tests/integrations/test_cases/execute.py

```diff
@@ -1353,10 +1353,10 @@
             "events": [
                 [
                     [
-                        0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF,
-                        0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF,
                         0xFF,
                         0x00,
+                        0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF,
+                        0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF,
```

Comment:
why do we have changes in test while the stack is, though refactor, meant to behave the same?

Reply:
Before pop[n] was returning, the "highest" stack index in popped[0] but now its the "lowest index". Basically it have changed the order of the array that we get back on popped, so all I did was to "reverse" the order of the test, in case for logging

Reply:
I've realized that pop was reversed, but AFAIU the LOG should not be impacted by implementation of the stack

Reply:
It is, because it pops "log_n + 2" using pop_n, before was popped[n] and popped[n-1] was the arguments and the rest log info, now the arguments are in popped[0] and popped[1] and the log info are from popped[2] -> popped[n]